docs/components/Header.tsx

```diff
@@ -0,0 +1,113 @@
+                            zero-knowledge proofs.
+                        </p>
+
+                        <div className="mb-8 transform transition-all duration-200 hover:scale-105 relative">
+                            <button
+                                onClick={handleCopy}
+                                className="w-full text-left group"
+                            >
+                                <pre className="p-5 rounded-xl bg-[#181818] backdrop-filter backdrop-blur-lg shadow-lg group-hover:shadow-red-600/5 duration-150 group-hover:bg-[#202020] transition-colors relative overflow-hidden overflow-x-auto flex justify-between items-center">
+                                    <code className="text-sm sm:text-base">
+                                        curl -L https://install.dojoengine.org |
+                                        bash
+                                    </code>
+                                    <svg
+                                        xmlns="http://www.w3.org/2000/svg"
+                                        width="16"
+                                        height="16"
+                                        viewBox="0 0 24 24"
+                                        fill="none"
+                                        stroke="currentColor"
+                                        strokeWidth="2"
+                                        strokeLinecap="round"
+                                        strokeLinejoin="round"
+                                        className="ml-4 opacity-50 group-hover:opacity-100 transition-opacity"
+                                    >
+                                        <rect
+                                            x="9"
+                                            y="9"
+                                            width="13"
+                                            height="13"
+                                            rx="2"
+                                            ry="2"
+                                        ></rect>
+                                        <path d="M5 15H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9a2 2 0 0 1 2 2v1"></path>
+                                    </svg>
+                                </pre>
+                            </button>
+                            {showCopied && (
+                                <div
+                                    className="absolute top-1/2 right-0 mr-4 transform -translate-y-1/2 px-3 py-1 bg-[#FF4136] text-white text-sm rounded-md opacity-0 transition-all duration-200 ease-in-out"
+                                    style={{ opacity: showCopied ? 1 : 0 }}
+                                >
+                                    Copied!
+                                </div>
+                            )}
+                        </div>
+
+                        <div className="flex gap-4">
+                            <Link
+                                to="/docs"
```

Comment:
```suggestion
                                to="/framework"
```
Or do you meant to add a `docs` page?